assets/src/adaptivity/capi.ts

```diff
@@ -59,6 +59,9 @@ export const coerceCapiValue = (
           );
         }
       }
+      if (typeof value === 'string') {
```

Comment:
we need to do this only in the scripting, this method is supposed to return the actual value, here it would be returning a sort of stringified version. the enum will always be a string already.